idea/testData/inspectionsLocal/collections/simplifiableCallChain/listOfNotNullWIthComment.kt

```diff
@@ -0,0 +1,8 @@
+// WITH_RUNTIME
+
+fun test() {
+    <caret>listOf(
```

Comment:
I think one example may be not enough here. At least, it would be good to test comment preserving for all three `when` branches above.